### ISSUE TITLE
VQA Score support

### ISSUE THREAD

**Issue by Dratlan** [QwenLM/Qwen2-VL#408]
Hi. I wonder can we get VQA Score from Qwen2-VL like the way in:
https://github.com/linzhiqiu/t2v_metrics
It seems that Qwen2-vl can only answer with 'yes' or 'no', but not score.

Many thanks!

**Comment by yhy-2000** [comment]
same question, when I try to output_scores/output_logits, the logits is -nan and all outputs turn to 'A'.
